recipes/maxit/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "MAXIT" %}
+{% set version = "11.200" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sw-tools.rcsb.org/apps/{{ name }}/{{ name|lower }}-v{{ version }}-prod-src.tar.gz
+  sha256: 658e236c6310cf7e55218a0500f82050ef86074f5d5b3f61d2a161b04a38cc39
+  patches:
+    - platform.sh.patch  # For osx-arm64
```

```suggestion
    - platform.sh.patch  # [osx and arm64]
```
This way the patch is applied only for OSX-ARM64.
Otherwise https://github.com/bioconda/bioconda-recipes/pull/50461/files#diff-0b9cbe761060d0c4a2a0404bfc4066f70c63eb4baac49dee8ec8ad35ad4d22b2R21 may fail on Linux

---

@martin-g 

This is `platform.sh` extended.

For Linux the patch is not needed because another case section handles it.

```shell
#   Check if it is a Mac OS platform
    Darwin)
#     First check if it is clang
      clang_ver=$(clang --version | grep -e "clang version")
      if [ ! -z "$clang_ver" ]
      then
        sysid="darwin4"
      else
#       Check if it is GCC version 4.x
        gcc_ver=$(gcc --version | grep -e " 4\.")
        if [ -z "$gcc_ver" ]
        then
#         It is not GCC version 4.x. Check if it is GCC version 3.x
          gcc_ver=$(gcc --version | grep -e " 3\.")
          if [ -z "$gcc_ver" ]
          then
#           It is not GCC version 3.x. Check if it is GCC version 2.x
            gcc_ver=$(gcc --version | grep -e "2\.")
            if [ -z "$gcc_ver" ]
            then
#             It is not GCC version 2.x either. Production can not be compiled.
              sysid="unknown"
            else
#             It is GCC version 2.x
              sysid="darwin2"
            fi
          else
#           It is GCC version 3.x
            sysid="darwin3"
          fi
        else
#         It is GCC version 4.x
          sysid="darwin4"
        fi
      fi
    ;;

#   Check if it is a Linux platform
    Linux)
#     Check if it is GCC version 11.x
      gcc_ver=$(gcc --version | grep -e " 11\.")
#     Check if it is GCC version 10.x
      if [ -z "$gcc_ver" ]
      then
          gcc_ver=$(gcc --version | grep -e " 10\.")
      fi
#     Check if it is GCC version 9.x
      if [ -z "$gcc_ver" ]
      then
          gcc_ver=$(gcc --version | grep -e " 9\.")
      fi
#     Check if it is GCC version 8.x
      if [ -z "$gcc_ver" ]
      then
          gcc_ver=$(gcc --version | grep -e " 8\.")
      fi
```

---

I understand that but I see no point to patch the shell script for non-OSX runs, thus my suggestion.

---

That is it, indeed.